runtime/browser/runtime_url_request_context_getter.cc

```diff
@@ -54,6 +57,24 @@ using content::BrowserThread;
 
 namespace xwalk {
 
+int GetDiskCacheSize() {
+  base::CommandLine* command_line = base::CommandLine::ForCurrentProcess();
+
+  if (!command_line->HasSwitch(switches::kDiskCacheSize))
+      return 0;
+
+  std::string str_value = command_line->GetSwitchValueASCII(
+      switches::kDiskCacheSize);
+
+  int size = 0;
+  if (!base::StringToInt(str_value, &size)) {
+      LOG(ERROR) << "The value " << str_value
+                 << " can not be converted to integer, ignoring!";
```

Comment:
I said before that exclamation marks should be avoided, but let's address that separately as its already taken so long.